main.ts

```diff
@@ -14,6 +17,7 @@ interface Settings {
 
 const DEFAULT_SETTINGS: Settings = {
 	vimrcFileName: ".obsidian.vimrc",
+	systemClipboard: true,

```

Comment:
I thought about building it into it, but the `vimrc` is read on every `file-open`. Monkey-patching is required once, and not every time. I didn't want to add reading of the `vimrc` file `onload` for that, a setting is better I think.

Reply:
A setting is definitely _simpler_, but I think it may lead to unpredictable behaviour for users.

Reply:
When users update, they'll get a notification that a the option is not recognised. I guess there needs to be something in the Changelog. If the maintainer doesn't write it, I could add that to the README.

Reply:
They do, but I'm thinking about the users who _don't_ have it set. They'll have their behaviour silently changed.

Reply:
I see, I didn't think about that. Do you think it would be enough to default to false? That way, it is non-destructive (the plugin won't write into the clipboard of users who don't have it set, everyone else will notice the change due to the message).

Reply:
I think defaulting to false would definitely solve the problem!

Reply:
Done: https://github.com/esm7/obsidian-vimrc-support/pull/107/commits/2e7d637f2e95baf208fdc96fae73048d7a58056b